api/utils/aws/identifiers.go

```diff
@@ -167,7 +167,7 @@ var (
 	//
 	// Reference:
 	// https://github.com/aws/aws-sdk-go-v2/blob/main/codegen/smithy-aws-go-codegen/src/main/resources/software/amazon/smithy/aws/go/codegen/endpoints.json
-	matchRegion = regexp.MustCompile(`^[a-z]{2}(-gov|-iso|-isob|-isoe)?-\w+-\d+$`)
+	matchRegion = regexp.MustCompile(`^[a-z]{2}(-gov|-global|-iso|-isob|-iso-b|-isoe|-iso-e|-isof|-iso-f)?-\w+-\d+$`)
```

Comment:
is my understanding correct that all global regions are the partition names with `-global` suffix? If so lets move `-global`/`-iso-b`/`-iso-e`/`-iso-f` out of this regex, and make a new function that checks is global region by checking `-global` suffix and whatever before it is a valid partition.

Reply:
I've removed the global regions matching since we're not including/using them in the other functions.